ios/browser/api/bookmarks/brave_bookmarks_observer.h

```diff
@@ -14,20 +14,26 @@ NS_ASSUME_NONNULL_BEGIN
 
 OBJC_EXPORT
 @protocol BookmarkModelObserver<NSObject>
+@optional
 - (void)bookmarkModelLoaded;
 
+@optional
 - (void)bookmarkNodeChanged:(BookmarkNode *)bookmarkNode;
 
+@optional
 - (void)bookmarkNodeChildrenChanged:
     (BookmarkNode *)bookmarkNode;
 
+@optional
 - (void)bookmarkNode:(BookmarkNode *)bookmarkNode
      movedFromParent:(BookmarkNode *)oldParent
             toParent:(BookmarkNode *)newParent;
 
+@optional
 - (void)bookmarkNodeDeleted:(BookmarkNode *)node
                  fromFolder:(BookmarkNode *)folder;
 
+@optional
 - (void)bookmarkModelRemovedAllNodes;
```

Comment:
You only need 1 `@optional`, all methods below it will be marked as optional :)

Reply:
You could say those extra `@optional` specifiers are OPTIONAL! 😂